python/mxnet/log.py

```diff
@@ -63,12 +63,38 @@ def format(self, record):
 def getLogger(name=None, filename=None, filemode=None, level=WARNING):
     """Get customized logger.
```

Comment:
'"Gets a" customer logger' or may be '"Returns a" customer logger' (preferred I think).